docs/bigquery/snippets.py

```diff
@@ -1097,100 +1108,37 @@ def test_load_table_from_uri_csv_append(client, to_delete):
 
     job_config = bigquery.LoadJobConfig()
     job_config.write_disposition = bigquery.WriteDisposition.WRITE_APPEND
+    # [END bigquery_load_table_gcs_csv_append]
+    # [END bigquery_load_table_gcs_json_append]
+    # [END bigquery_load_table_gcs_parquet_append]
+
+    # Format-specific code
+    # [START bigquery_load_table_gcs_csv_append]
     job_config.skip_leading_rows = 1
     # The source format defaults to CSV, so the line below is optional.
     job_config.source_format = bigquery.SourceFormat.CSV
-
-    load_job = client.load_table_from_uri(
-        'gs://cloud-samples-data/bigquery/us-states/us-states.csv',
-        table_ref,
-        job_config=job_config)  # API request
-
-    assert load_job.job_type == 'load'
-
-    load_job.result()  # Waits for table load to complete.
-
-    assert load_job.state == 'DONE'
-    assert client.get_table(table_ref).num_rows == previous_rows + 50
+    uri = 'gs://cloud-samples-data/bigquery/us-states/us-states.csv'
     # [END bigquery_load_table_gcs_csv_append]
-
-    assert previous_rows == 1
-
-
-def test_load_table_from_uri_json_append(client, to_delete):
-    dataset_id = 'load_table_dataset_{}'.format(_millis())
-    dataset = bigquery.Dataset(client.dataset(dataset_id))
-    dataset.location = 'US'
-    client.create_dataset(dataset)
-    to_delete.append(dataset)
-
-    job_config = bigquery.LoadJobConfig()
-    job_config.schema = [
-        bigquery.SchemaField('name', 'STRING'),
-        bigquery.SchemaField('post_abbr', 'STRING')
-    ]
-    table_ref = dataset.table('us_states')
-    body = six.BytesIO(b'Washington,WA')
-    client.load_table_from_file(
-        body,
-        table_ref,
-        location='US',  # Location must match that of the destination dataset.
-        job_config=job_config).result()
+    # unset csv-specific attribute
+    del job_config._properties['load']['skipLeadingRows']
```

Comment:
Could we set the public property to `None` to accomplish the same thing?

Reply:
That was the first thing I tried, but it tried to send `None` to the API and it sent back an error. The whole `_properties` dict gets sent to the API, so I had to delete it to prevent that.

Reply:
Okay, makes sense. We could potentially add some property deleters, but let's wait until someone has a real usecase for one.